ext/mysqli/mysqli.c

```diff
@@ -820,11 +718,6 @@ PHP_MSHUTDOWN_FUNCTION(mysqli)
 /* {{{ PHP_RINIT_FUNCTION */
 PHP_RINIT_FUNCTION(mysqli)
 {
-#if !defined(MYSQLI_USE_MYSQLND) && defined(ZTS)
```

Comment:
Maybe should still be ``#ifdef ZTS``?

Reply:
It was logical and. I don't think we need it as it would be `if false && defined(ZTS)`. Am I missing something?

Reply:
@kamil-tekiela, yeah, you're right.